witchcraft/refreshable/refreshable_file_test.go

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2018 Palantir Technologies. All rights reserved.
```

Comment:
nit: since this file was newly created, copyright year should be 2020

Reply:
Done